pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+
+addopts = --doctest-modules -v --ignore=exclude
```

Comment:
this goes in the pyproject.toml -> https://github.com/facastagnini/python-audiobook-tools/blob/main/pyproject.toml#L5-L6